specs/Qowaiv.Specs/Date_span_specs.cs

```diff
@@ -107,3 +108,145 @@ public void with_info()
            format: "date-span",
            pattern: @"[+-]?[0-9]+Y[+-][0-9]+M[+-][0-9]+D"));
 }
+
+public class Can_be_operated
+{
+    [Test]
+    public void negate_negates_values()
+    {
+        var negated = -Svo.DateSpan;
+        negated.Should().Be(new DateSpan(-10, -3, +5));
+    }
+
+    [Test]
+    public void plus_does_nothing()
+    {
+        var plussed = +Svo.DateSpan;
+        plussed.Should().Be(Svo.DateSpan);
+    }
+}
+
+public class Can_create
+{
+    [Test]
+    public void Age_form_date_without_months()
+    {
+        using (Clock.SetTimeForCurrentContext(() => new Date(2019, 10, 10)))
```

Comment:
Suggest use [UseTime] Attribute

Reply:
That option does not exist in this repo. ;)